streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -1671,8 +1671,8 @@ class ZStream[-R, +E, +A](val process: ZManaged[R, E, Pull[R, E, A]]) extends Se
    * Repeats are done in addition to the first execution, so that `scheduleElements(Schedule.once)` means "emit element
    * and if not short circuited, repeat element once".
    */
-  final def scheduleElements[R1 <: R, A1 >: A](schedule: ZSchedule[R1, A, A1]): ZStream[R1 with Clock, E, A1] =
-    scheduleElementsEither(schedule).map(_.merge)
+  final def scheduleElements[R1 <: R, A1 >: A](schedule: ZSchedule[R1, A, _]): ZStream[R1 with Clock, E, A1] =
```

Comment:
I'd use `Any` instead of `_`, as existentials are going away in Dotty.

Reply:
Hmm, let's open an issue that checks for that. I have seen existentials in other places as well.